.gitignore

```diff
@@ -5,7 +5,7 @@ bid/development-bundle
 bid/backup/
 content/cache/*
 dbconfig.inc
-dbconfig.py
+dbconfig.json
```

Comment:
So, we also need to submit a dbconfig.inc to the repo that reads from the json file too, since the json is now where the secrets are.